pkg/sloop/queries/timerange_test.go

```diff
@@ -55,8 +59,8 @@ func Test_timeRangeTable(t *testing.T) {
 		{"", "abc", "123", true, time.Time{}, time.Time{}},
 		{"", "123", "abc", true, time.Time{}, time.Time{}},
 	}
-	for _, thisRange := range rangeTests {
-		t.Run(fmt.Sprintf("%+v", thisRange), func(t *testing.T) {
+	for i, thisRange := range rangeTests {
+		t.Run(fmt.Sprintf("Test id: %v, thisRange: %v", i, thisRange), func(t *testing.T){
```

Comment:
nit: just curious why did we change `%+v` to `%v`?

Reply:
you are correct, change back to `%+v` as plus flag adds field names